cvat-ui/src/components/create-task-page/project-search-field.tsx

```diff
@@ -12,6 +12,7 @@ const core = getCore();
 
 type Props = {
     value: number | null;
+    excludeId?: number;
```

Comment:
Another possible option (more general I believe) is to pass optional filter() callback.

Reply:
done